src/main/scala/com/github/swagger/enumeratum/converter/SwaggerEnumeratumModelConverter.scala

```diff
@@ -97,4 +97,3 @@ class SwaggerEnumeratumModelConverter extends ModelResolver(Json.mapper()) {
     case Some(arr) => arr.toList
   }
```

Comment:
there are no src/main changes - all this PR seems to do is add an extra assertion

Reply:
This PR is the `swagger-scala-module` counterpart https://github.com/swagger-akka-http/swagger-scala-module/pull/211
It contains the fix for the issue reproduced here.